aveas_openlabel/attributes/summary.py

```diff
@@ -258,3 +258,14 @@ class Summary__Coordinates__ScenarioEnd__UStdDev(VectorData):
 
     name: Literal["summary/coordinates/scenario_end/ustddev"] = field(default="summary/coordinates/scenario_end/ustddev")
     """Is always 'summary/coordinates/scenario_end/ustddev'."""
+
+
+@dataclass
+class Summary__Frame__Appearance(VectorData):
```

Comment:
This is already covered by the OpenLABEL frame intervals. Why define an attribute for it?